src/airflow_declarative/schema.py

```diff
@@ -120,10 +122,13 @@ def represent_callable(self, obj):
 CLASS = Class()
 DATE = Date()
 EMAIL = Email
+NULL = Null()
 POSITIVE_INT = Int(gte=0)
 STRING = String()
 TIMEDELTA = TimeDelta()
 
+AIRFLOW_NULL = NULL | Regexp(regexp="None")
```

Comment:
I believe it should be `^None$`. And it's true that there are no way to match a static string without regexp?

Reply:
Enum seems to be a viable alternative: https://github.com/Deepwalker/trafaret/blob/40a521ae0f580a0e5a6f3168a59cbba350d5b3ba/trafaret/base.py#L1198-L1222

But I have a question: why would we want to allow `None` as string instead of accepting yaml's `null` which would be turned to the Python's `None` by yaml parser?

Reply:
Can do it with Enum. I allow None as string, because airflow allow None and 'None'. Both this works, for this PR, but need to add test for null.

Reply:
That is a rather strange decision from the Airflow side. Anyways, we don't have to add support for `"None"`. I would suggest to allow only the YAML's `null`. If the behavior is the same as for `"None"`, that should be enough. Having less ways to achieve the same is usually better.

Reply:
Drop support for 'None' as string.